File: RemoveRecursionRefactoring/tests/ro/pub/cs/diploma/ReplaceSingleStatementsWithBlockStatementsFixTest.java
package ro.pub.cs.diploma;

import ro.pub.cs.diploma.inspections.ReplaceSingleStatementsWithBlockStatementsInspection;

public class ReplaceSingleStatementsWithBlockStatementsFixTest extends IGQuickFixesTestCase {
  @Override
  protected void setUp() throws Exception {
    super.setUp();
    myFixture.enableInspections(new ReplaceSingleStatementsWithBlockStatementsInspection());
    myRelativePath = "replace-single-statements-with-block-statements";
    myDefaultHint = RemoveRecursionBundle.INSTANCE.message("replace.single.statements.with.block.statements");
  }

  public void testReplaceSingleStatementsWithBlockStatements() {
    doTest();
  }
}

File: inspection/src/test/kotlin/ro/pub/cs/diploma/ReplaceSingleStatementsWithBlockStatementsFixTest.kt
package ro.pub.cs.diploma

import org.junit.Ignore
import ro.pub.cs.diploma.inspections.ReplaceSingleStatementsWithBlockStatementsInspection

@Ignore
class ReplaceSingleStatementsWithBlockStatementsFixTest : IGQuickFixesTestCase() {
    @Throws(Exception::class)
    override fun setUp() {
        super.setUp()
        myFixture.enableInspections(ReplaceSingleStatementsWithBlockStatementsInspection())
        myRelativePath = "replace-single-statements-with-block-statements"
        myDefaultHint = RemoveRecursionBundle.message("replace.single.statements.with.block.statements")
    }

    fun testReplaceSingleStatementsWithBlockStatements() {
        doTest()
    }
}
